modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
I'm not sure I understand. Are you just setting `packageName` to "IO.Swagger" and "MyProject.Api"? Or are you changing `apiPackage` and `modelPackage`?

You'd set the api and model packages/namespaces with an additional switch, for example:

```
--additional-properties=apiPackage=PublicApi
```

And in this case, the modified glob patterns will cause the API classes to not be compiled in the new code.